src/ImageInTerminal.jl

```diff
@@ -39,10 +39,9 @@ Call `ImageInTerminal.use_256()` to restore default behaviour.
 use_24bit() = (colormode[1] = TermColor24bit())
 
 # colorant arrays
-function Base.show(
```

Comment:
Although not changing the numerical results, deleting `Base.show` still makes this PR a breaking change

Reply:
ok, just a quick suggestion, fewest lines changed!

Reply:
I restored the show `method`, so that this `print_matrix` is called only in the case of 3-dim & higher arrays.